bokehjs/src/coffee/core/layout/layout_canvas.coffee

```diff
@@ -0,0 +1,49 @@
+_ = require "underscore"
+
+{Variable, EQ, GE, Strength}  = require "./solver"
+Model = require "../../model"
+p = require "../properties"
+
+class LayoutCanvas extends Model
+  type: 'LayoutCanvas'
+
+  @internal {
```

Comment:
Perhaps related is that I'd like `LayoutDOM` (or whatever it's to be called) to also be private and not exposed to users and to be put into `/core/layout` in the same place as `LayoutCanvas` but it seemed to be necessary to make it public....I think because of the check in `io.py` - if you have any suggestions, in this area, that would be great.